sdk/search/azure-search-documents/pom.xml

```diff
@@ -149,5 +149,28 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>parallel-test-playback</id>
+      <activation>
+        <property>
+          <name>parallel-test-playback</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>3.0.0-M3</version> <!-- {x-version-update;org.apache.maven.plugins:maven-surefire-plugin;external_dependency} -->
+            <configuration>
+              <systemPropertyVariables>
```

Comment:
Does it affect live tests? Why search is disabled?

Reply:
When using the host agent parallel runs will only happen in PR validation, so this won't affect live tests. I updated the PR description to explain that this was done since playback is much safer to parallelize without an in-depth investigation due to the locality of testing resources, if you want to you could run tests in parallel locally against the service.

Parallel tests for Search has been disabled as there is some testing global state that doesn't play well with parallelization and causes intermittent test failures, I added a comment into the POM stating this.